pkg/pillar/cmd/zedagent/parseconfig.go

```diff
@@ -711,12 +710,11 @@ func parseOneSystemAdapterConfig(getconfigCtx *getconfigContext,
 	if sysAdapter.Addr != "" {
 		ip = net.ParseIP(sysAdapter.Addr)
 		if ip == nil {
-			errStr := fmt.Sprintf("parseSystemAdapterConfig: Port %s has Bad "+
+			err := fmt.Errorf("parseSystemAdapterConfig: Port %s has Bad "+
 				"sysAdapter.Addr %s - ignored",
 				sysAdapter.Name, sysAdapter.Addr)
-			log.Error(errStr)
-			port.ParseError = errStr
-			port.ParseErrorTime = time.Now()
+			log.Error(err)
+			port.SetError(err, "zedagent parseconfig")
```

Comment:
These errors will be user-visible, hence should be actionally. I'd argue that the old message string "parseSystemAdapterConfig:"  doesn't help explain the issue and how the user should resolve it. But adding a "zedagent parseconfig" to the string makes it worse.

Reply:
Let me change the Error message to:

"Device Config Error.  Port %s has Bad sysAdapter.Addr %s. The Adapter is ignored. Please fix the device configuration."

Change the source of the error to "Config Sanity Check"

log.Error --> I will change to print the parseSystemAdapterConfig in addition to the error..